src/diffpy/labpdfproc/tests/test_tools.py

```diff
@@ -105,16 +104,33 @@ def test_load_user_metadata(inputs, expected):
 
 
 params6 = [
-    ([["facility=NSLS", "II"]]),
-    ([["favorite", "color=blue"]]),
-    ([["beamline", "=", "28ID-2"]]),
+    (
+        [["facility=NSLS", "II"]],
+        [
+            "Please provide key-value pairs in the format key=value. "
+            "For more information, use `labpdfproc --help.`"
+        ],
+    ),
+    (
+        [["favorite", "color=blue"]],
+        "Please provide key-value pairs in the format key=value. "
+        "For more information, use `labpdfproc --help.`",
+    ),
+    (
+        [["beamline", "=", "28ID-2"]],
+        "Please provide key-value pairs in the format key=value. "
+        "For more information, use `labpdfproc --help.`",
+    ),
+    ([["facility=NSLS II", "facility=NSLS III"]], ["Please do not specify repeated keys: facility. "]),
+    ([["wavelength=2"]], ["Please do not specify repeated keys: wavelength. "]),
```

Comment:
This one might be confusing because the user didn't specify repeated keys.  maybe `wavelength is a reserved name.  Please rerun using a different key name`?

We may want to store a list of reserved key names (it will basically  be `vars(args).keys()`).  We could list this in the args help if we want.